about.html

```diff
@@ -40,13 +40,14 @@
         <section class="about-page hero-section d-flex" id="Home">
             <div class="section-container d-flex">
                 <div class="title-box d-flex">
+                    <div class="separator"></div>
                     <h2 class="subtitle bold">Summit 2023</h2>
                     <h1 class="title bold">Leaders Week</h1>
                 </div>
                 <p class="description">
                     A joyful celebration in the value of openess and sharing,
                     creating a positive change with people from over 10 countries
-                    is taking place in July, in Benin.
+                    is taking place in July, in Cotonou, Benin.
```

Comment:
- [ ] **Good job so far 👏** Kindly as suggested by the previous reviewer, it will be better if you fix the responsiveness of your page as the elements overwrap on the screen width of `439` to `375`, It would be better to make this small adjustment for the best user experience 🙏 

Check the video below for your reference 😊⤵️
https://www.loom.com/share/723bf826dca946bdad3e6a329a33f86f

If you can not see the video clearly just click on it to open a new tab in the browser😊

Reply:
Thanks 🙏 @TuyishimireEric for reviewing my pull request.
I appreciate your review I will work on these parts to enhance my code.
Have a good day!